exercises/finale-workflow/Client/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Temporalio.Client;
+using Temporalio.Finale.Workflow;
+
+var client = await TemporalClient.ConnectAsync(new ()
+{
+    TargetHost = "localhost:7233",
+});
+
+// TODO: Change 'Maxim Fateev' to your name
+var handle = await client.StartWorkflowAsync((CertificateGeneratorWorkflow wf) => wf.RunAsync("Maxim Fateev"),
+    new WorkflowOptions
+    {
+        TaskQueue = "generate-certificate-taskqueue",
+        Id = "cert-generator-workflow-" + Guid.NewGuid()

```

Comment:
```suggestion
        Id = $"cert-generator-workflow-{Guid.NewGuid()}"
```

Other PRs are doing string interpolation